utils/test/utils_test.exs

```diff
@@ -73,6 +73,24 @@ defmodule UtilsTest do
     end
   end
 
+  test "Ensure no broken / empty links in livebooks" do
+    exercises = fetch_livebooks("../exercises/")
+    reading = fetch_livebooks("../reading/")
+
+    assert any_invalid_links?(~r/\]\(.*exercises\/\w+.livemd\)/, exercises) == false
```

Comment:
I think it's possible for exercises to contain links to reading files and vice versa 🤔 i.e. `../reading/reading.livemd`.

Not sure if there are any, though. What do you think, is it worth adding to the test?